src/script/lua_api/l_env.cpp

```diff
@@ -340,6 +340,35 @@ int ModApiEnv::l_swap_node(lua_State *L)
 	return 1;
 }
 
+// bulk_swap_node([pos1, pos2, ...], node)
+// pos = {x=num, y=num, z=num}
+int ModApiEnv::l_bulk_swap_node(lua_State *L)
+{
+	GET_ENV_PTR;
+
+	luaL_checktype(L, 1, LUA_TTABLE);
+
+	s32 len = lua_objlen(L, 1);
+	if (len == 0) {
+		lua_pushboolean(L, true);
+		return 1;
+	}
```

Comment:
I think this should also be removed because it's not a relevant optimization and introduces another edge case where the second argument is not type-checked if the first one is an empty list

```suggestion
```